src/main/java/PersonalBudget/business/expense/domain/model/ExpenseCategoryEntity.java

```diff
@@ -0,0 +1,36 @@
+package PersonalBudget.business.expense.domain.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.SequenceGenerator;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table(name = "expense_category")
+public class ExpenseCategoryEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "expense_category_seq")
```

Comment:
You should name your sequence generator differently as the sequence.
```suggestion
    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "expense_category_sg")
```